backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -355,7 +357,7 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
         initializeCachedBoxes(context)
         declaration.acceptChildrenVoid(this)
 
-        runAndProcessInitializers(declaration.descriptor) {
+        runAndProcessInitializers(declaration.konanLibrary) {
```

Comment:
Actually `konanLibrary` is probably always `null` here, and it makes a lot of sense to use `null` here if this is true.